SES_AND_LAVAMOAT.md

```diff
@@ -0,0 +1,21 @@
+# LavaMoat and SES (Secure Ecmascript) maintenance
+
+### Hermes
+
+SES does not yet work with Hermes, so we had to disable it on both platforms https://github.com/facebook/hermes/issues/957
+
+### SES injection
+
+LavaMoat does not yet fully support react-native, there is a compatibility tracker [here](https://github.com/LavaMoat/docs/issues/12). Until then, we're using [SES](https://github.com/endojs/endo/tree/master/packages/ses) directly, injecting it directly into the InitializeCore which is the entrypoint for the react-native's bundle so it hardens our app as soon as possible in the process.
+
+This is done through a patch on the react-native package, using `patch-package`.
+
+We are currently using version `1.0.1` of SES, which is the latest version that works with react-native, it is loaded from the project root, `lockdown.umd.js`.
```

Comment:
Is this still correct? It's been 6 months since you did it.

Reply:
Updated this document with a better explanation on SES